cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/AbstractStorageProviderNamespaceAdmin.java

```diff
@@ -69,7 +69,8 @@ public void create(NamespaceMeta namespaceMeta) throws IOException, ExploreExcep
 
     createLocation(namespaceMeta);
 
-    if (cConf.getBoolean(Constants.Explore.EXPLORE_ENABLED)) {
+    if (cConf.getBoolean(Constants.Explore.EXPLORE_ENABLED) &&
+      !NamespaceId.DEFAULT.equals(namespaceMeta.getNamespaceId())) {
```

Comment:
What is this about? Why the DEFAULT is a special case? Might be good to add a comment as well as in the Javadoc.